src/cdef.rs

```diff
@@ -55,9 +56,12 @@ wrap_fn_ptr!(pub unsafe extern "C" fn cdef(
     damping: c_int,
     edges: CdefEdgeFlags,
     bitdepth_max: c_int,
-    _dst: *const FFISafe<Rav1dPictureDataComponentOffset>,
-    _top: *const FFISafe<CdefTop>,
-    _bottom: *const FFISafe<CdefBottom>,
+    dst: *const FFISafe<Rav1dPictureDataComponent>,
+    dst_offset: usize,
+    top: *const FFISafe<DisjointMut<AlignedVec64<u8>>>,
+    top_offset: usize,
+    bottom: *const FFISafe<PicOrBuf<'_, AlignedVec64<u8>>>,
+    bottom_offset: usize,
```

Comment:
```suggestion
    _dst: *const FFISafe<Rav1dPictureDataComponent>,
    _dst_offset: usize,
    _top: *const FFISafe<DisjointMut<AlignedVec64<u8>>>,
    _top_offset: usize,
    _bottom: *const FFISafe<PicOrBuf<'_, AlignedVec64<u8>>>,
    _bottom_offset: usize,
```

As a convention for readability, I've been prefixing the extra args not used by asm with a `_`.